src/run.ts

```diff
@@ -15,9 +15,16 @@ export const run = (context: Context) => {
   info(`Pull Request title: "${pullRequestTitle}"`);
 
   const regex = RegExp(getInput("regexp"));
+  const helpMessage = getInput("helpMessage");
   if (!regex.test(pullRequestTitle)) {
-    setFailed(
-      `Pull Request title "${pullRequestTitle}" failed to pass match regex - ${regex}`
-    );
+    let message = `Pull Request title "${pullRequestTitle}" failed to pass match regexp
+Used Regexp: ${regex}
```

Comment:
Not sure the new line clarifies the log, I think I preferred it before. Do you think it's easier to read this way?

Reply:
As you prefer :)